### PR TITLE
DMP-1437 - Check your retention date change screen

### PR DESCRIPTION
**Pull request by joshw-hmcts**
### Jira link (if applicable)

[DMP-1437](https://tools.hmcts.net/jira/browse/DMP-1437)

### Change description ###

Check retention date change screen built which shows after confirming date and reason on Change case retention date screen

### Checklist

- [X] commit messages are meaningful and follow good commit message guidelines
- [X] README and other documentation has been updated / added (if needed)
- [X] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: src/app/components/case/case-retention-date/case-retention-date.component.html [hunk 1]
```diff
@@ -80,14 +80,28 @@ <h2 class="govuk-heading-m">Retention audit history</h2>
...
       [originalRetentionDate]="getOriginalRetentionDateString(vm.retentionHistory)"
       (stateChange)="onStateChanged($event)"
+      (retentionDateChange)="onRetentionDateChanged($event)"
+      (retentionReasonChange)="onRetentionReasonChanged($event)"
     ></app-case-retention-change>
   </div>
+  <div *ngIf="state === 'Change' || state === 'Confirm'" [hidden]="state === 'Change'" class="order-confirmation">
+    <app-case-retention-confirm
+      [state]="state"
+      [newRetentionDate]="newRetentionDate"
```
Comment (DavidTMann) [inline src/app/components/case/case-retention-date/case-retention-date.component.html:96]: this is duplicated

**Review by bpires2 — APPROVED** [review]

File: src/app/components/case/case-retention-date/case-retention-date.component.html (hunk 1 above)
Comment (joshw-hmcts) [reply src/app/components/case/case-retention-date/case-retention-date.component.html:96]: Fixed

File: src/app/components/case/case-retention-date/case-retention-date.component.html [hunk 2]
```diff
@@ -80,28 +80,30 @@ <h2 class="govuk-heading-m">Retention audit history</h2>
...
+      ></app-case-retention-change>
+    </div>
+    <div [hidden]="state === 'Change'" class="order-confirmation">
+      <app-case-retention-confirm
+        [state]="state"
+        [newRetentionDate]="newRetentionDate"
+        [caseId]="vm.caseDetails.details['Case ID']"
+        [caseCourthouse]="vm.caseDetails.details['Courthouse']"
+        [caseDefendants]="vm.caseDetails.details['Defendant(s)']"
+        [newRetentionDate]="newRetentionDate"
```
Comment (DavidTMann) [inline src/app/components/case/case-retention-date/case-retention-date.component.html:101]: still duped

Comment (joshw-hmcts) [reply src/app/components/case/case-retention-date/case-retention-date.component.html:100]: Removed for real this time

**Pull request merged**